src/methods/trackTransactions/helpers/checkTransactionStatus/runSessionCallbacks.ts

```diff
@@ -1,30 +1,30 @@
-import { sessionCallbacksMap } from 'managers/TransactionManager/helpers/sessionCallbacks';
+import { getCallbacks } from 'managers/TransactionManager/helpers/sessionCallbacks';
 import { TransactionBatchStatusesEnum } from 'types/enums.types';
 
-interface runSessionCallbacksPropsType {
+interface RunSessionCallbacksPropsType {
   sessionId: string;
   status?: TransactionBatchStatusesEnum | null;
 }
 
 export async function runSessionCallbacks({
   sessionId,
   status
-}: runSessionCallbacksPropsType) {
-  const { onSuccess, onFail } = sessionCallbacksMap;
+}: RunSessionCallbacksPropsType) {
+  const callbacks = getCallbacks(sessionId);
 
-  if (!onSuccess || !onFail || status == null) {
+  if (!callbacks.onSuccess || !callbacks.onFail || status == null) {
```

Comment:
!callbacks?.onSuccess || !callbacks?.onFail, because at line 21 and line 27 you use await callbacks?.onSuccess?. and await callbacks?.onFail?.